src/main/antora/modules/archive/pages/jakarta-tags/jakarta-tags.adoc

```diff
@@ -0,0 +1,3 @@
+= Jakarta-Tags
+
+[TBD]
```

Comment:
It's OK to add links to

- https://docs.oracle.com/javaee/5/tutorial/doc/bnakq.html
- https://docs.oracle.com/javaee/5/tutorial/doc/bnakw.html
- https://docs.oracle.com/javaee/5/tutorial/doc/bnald.html